docs/QuickStart.md

```diff
@@ -1,64 +1,101 @@
-
-Table of Contents
-=================
-
-  * [Refocus Quick Start Guide](#refocus-quick-start-guide)
-    * [Why use Refocus?](#why-use-refocus)
-  * [Quick Start](#quick-start)
-    * [Step 1: Use the API to create your Subjects](#step-1-use-the-api-to-create-your-subjects)
-      * [Create a root Subject](#create-a-root-subject)
-      * [Create a child Subject](#create-a-child-subject)
-      * [Create another child Subject](#create-another-child-subject)
-    * [Step 2: Use the API to create your Aspects.](#step-2-use-the-api-to-create-your-aspects)
-    * [Step 3: Use the API to send some samples.](#step-3-use-the-api-to-send-some-samples)
-    * [Step 4: Install a lens](#step-4-install-a-lens)
-    * [Step 5: Create a perspective](#step-5-create-a-perspective)
-    * [Step 6: Refocus!](#step-6-refocus)
+# Refocus 
+*Quick Start Guide*
+
+  * [Why Use Refocus?](#why-use-refocus)
+  * [Refocus Terminology](#refocus-terminology)
+  * [Refocus Interface](#refocus-interface)
+  * [Getting Started with Refocus](#getting-started-with-refocus)
+    * [Refocus API](#refocus-api)
+    * [Step 1: Use the API to create your subject.](#step-1-use-the-api-to-create-your-subject)
+      * [Create a Root Subject](#create-a-root-subject)
+      * [Create a Child Subject](#create-a-child-subject)
+      * [Create Another Child Subject](#create-another-child-subject)
+    * [Step 2: Use the API to create aspects.](#step-2-use-the-api-to-create-aspects)
+    * [Step 3: Use the API to send a sample.](#step-3-use-the-api-to-send-a-sample)
+    * [Step 4: Install a lens.](#step-4-install-a-lens)
+    * [Step 5: Create a perspective.](#step-5-create-a-perspective)
+    * [Step 6: Start using Refocus.](#step-6-start-using-refocus)
     * [Advanced Topics](#advanced-topics)
       * [Related Links](#related-links)
-      * [Coming Soon: Auth Tokens](#coming-soon-auth-tokens)
-      * [Create your own lens](#create-your-own-lens)
-  * [Thank You!](#thank-you)
-
+      * [Authorization Tokens Are Coming Soon!](#authorization-tokens-are-coming-soon)
 
 ## Why use Refocus?
 
-If you care about how your service is performing _right now_, use Refocus to:
-- connect disparate monitoring tools into a self-service, unified platform
-- visualize your service health beautifully and unambiguously
+Refocus is a visualization platform for the status and health of anything you want to monitor.
+If you care about how your service is performing *right now*, use Refocus to:
+- **Understand**  
+  Understand your data in new ways by gaining insight
+  
+- **Connect**  
+  Connect various monitoring tools together into one unified platform
+  
+- **Visualize**  
+  Visualize your system health in beautiful and unambiguous ways
+
+## Refocus Terminology
+Let’s familiarize you with the terms we use in Refocus.
```

Comment:
Having the terminology live here in this doc is the best way to keep it current. Also, I'll change the coding so the terms are heading, which can then be linked (like the TOC).